es6-shim.js

```diff
@@ -1850,7 +1850,7 @@
       if (x === -1) { return -Infinity; }
       if (x === 1) { return Infinity; }
       if (x === 0) { return x; }
-      return 0.5 * _log((1 + x) / (1 - x));
+      return Math.sign(x) * Math.log1p(2 * Math.abs(x) / (1 - Math.abs(x))) / 2;
```

Comment:
yes, true - I'm ok with you punting on that for now tho, and i can take care of it all at once in a followup